README.md

```diff
@@ -10,14 +10,26 @@ parameterize a configuration to create reusable pipelines.
 - <b>example</b>: A practical example shown for each module loader plus without modules for comparison
 - <b>util</b>: Utilities for managing modules in this repo
 
-## Categories
-
-### OTLP
-
-OTLP covers multi use OpenTelemetry modules
+## Modules
 
 | Name |  Description | Agent Version | 
 | ---- |  ----------- | ------------- | 
 | [OTLP to LGTM](./modules/otlp/otlp-to-lgtm/) | Module to ingest OTLP data and then send it to Loki, Mimir and Tempo stacks locally or in GrafanaCloud. | `>= v0.33`
 | [Grafana Agent Telemetry to LGTM](./modules/grafana-agent/telemetry-to-lgtm/) | Module to forward the Grafana Agent's own telemetry data to Loki, Mimir and Tempo stacks locally or in Grafana Cloud. | `>= v0.33`
 | [Grafana Cloud Autoconfigure](./modules/grafana-cloud/autoconfigure/) | Module to automatically configure receivers for Grafana Cloud. | `>= v0.34`
+| [Host Filtering](./modules/host-filter/) | The host filtering module provides a Flow mode equivalent to static mode's host filtering functionality. | `>= v0.34`
+
+## Submitting modules
+
+Modules should be added under the `modules` folder. Each module should have a README.MD that has:
+* Name
+* Brief description
+* Applicable Agent Versions
+* Arguments
+* Exports
+* Example
+
+Module contents should be ordered:
+* `arguments`
+* `exports`
+* body of the module
```

Comment:
```suggestion
Modules must contain the following elements:
* Arguments
* Exports
* The body of the module

```

I removed the code markdown for arguments and exports since we are referring to them in general vs specifics. Order is important? Critical? Will things work if someone reverses the definitions for arguments and exports?

Do we provide a module content template for people to follow/fill in?

Reply:
Nothing will break, its a suggestion. We should create a template, I can work on that here in a few.